README.md

```diff
@@ -38,9 +34,6 @@ Responsibilities include:
 * Ensure the vulnerability data is updated in an efficient and timely manner. For example, ensuring there
   are well-documented processes for reporting vulnerabilities in community
   modules.
-* Define and maintain policies and procedures for the coordination of security
-  concerns within the external Node.js open source ecosystem.
-* Offer help to npm package maintainers to fix high-impact security bugs.
```

Comment:
This would need to be reflected in https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#ecosystem-security as well.